scripts/quests/otherAreas/Rycharde_the_Chef.lua

```diff
@@ -0,0 +1,146 @@
+-----------------------------------
+-- Rycharde the Chef
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/npc_util')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.RYCHARDE_THE_CHEF)
+
+quest.reward =
+{
+    fame  = 120,
+    title = xi.title.PURVEYOR_IN_TRAINING,
+    gil   = 1500,
+}
+
+quest.sections =
+{
+    -- Section: Steps prior to getting the quest.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.MHAURA] =
+        {
+            ['Rycharde'] =
```

Comment:
This could probably use the quest:event() function to simplify, as well as below for Take.